tensorboard/tools/do_not_submit_test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Since this just tripped me up, I'll observe that grep's `-I` and `--exclude-dir` flags don't appear to be in the POSIX spec for grep, so if Travis ever changed to use a POSIX-only shell for /bin/sh this might fail.  (The license test uses bash for its shebang.)

Reply:
Yes, `-I` and `--exclude-dir` aren’t specified by POSIX. POSIX `grep`,
`find`, and `xargs` are really anemic, unfortunately: `xargs` doesn’t
even have `-0`! But this is not a property of the shell; it is a
property of the implementation of `grep(1)` (contrast with `shift`,
which is and must be a shell builtin). The shebang is not relevant.

By using `-I` and `--exclude-dir`, we’re not assuming bash or any
particular shell; we’re assuming a `grep` implementation that supports
these options. GNU `grep` and BSD `grep` both support these options.

Reply:
Ah yes, of course it's not affected by the shell, duh.